src/assets/json/checklist/tests-ios-en.json

```diff
@@ -0,0 +1,155 @@
+	"profils": ["Checklist initiale"],
+	"wcag": ["2.1.1"],
+	"ID": "testMobID-4",
+	"IDorigin": "testMobID-4"
+}, {
+	"themes": "Text size",
+	"title": "Allow text increase",
+	"type": ["Manual test"],
+	"tests": ["Increase text size to 235%: <ul><li>Either through the Settings menu > Accessibility > Larger text, then move the slider to the 10th level.</li><li>Or by enabling zoom to 235% from the control center. See <a target=_blank href=https://a11y-guidelines.orange.com/en/articles/how-to-test-ios/>the article on getting started with features on iOS (new tab)</a>.</li></ul>"],
+	"verifier": ["The text enlarges correctly, without overlap and without loss of information (truncated or hidden content), and there are no unusable features."], 
+	"resultat": ["Text is easily readable.","No truncated or hidden content.","No unusable features."], 
+	"exception": "Elements in the header (navbar) and footer (tabbar) may not enlarge.",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/mobile/ios/development/#text-size",
+	"profils": ["Checklist initiale"],
+	"wcag": ["1.4.4"],
+	"ID": "testMobID-5",
+	"IDorigin": "testMobID-5"
+}, {
+	"themes": "Forms",
+	"title": "Make sure each form field has a visible and consistent label",
+	"type": ["Manual test"],
+	"tests": ["Identify the form elements in the application (input fields, checkboxes, radio buttons, toggle switches)."],
+	"verifier": ["All these elements have a visible label nearby.", " Note: the placeholder commonly found in input fields is not considered a label."],
+	"resultat": ["Each form element has a visible and consistent label nearby."],
+	"exception": "The label may be visually hidden when the role of the form field is clear and unambiguous (e.g., a search field accompanied by a magnifying glass button, a checkbox to select a row in a data table).",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/mobile/ios/design/#form",
+	"profils": ["Checklist initiale"],
+	"wcag": ["2.4.6", "3.3.2"],
+	"ID": "testMobID-6",
+	"IDorigin": "testMobID6"
+}, {
+	"themes": "Forms",
+	"title": "Make sure error messages are relevant",
+	"type": ["Manual test"],
+	"tests": ["Browse the forms.", "Fill in the forms with incorrect data.", "Submit the forms."],
+	"verifier": ["Error messages are present and relevant."],
+	"resultat": ["Error are detected and messages are relevant."],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/forms/#detect-identify-errors-and-suggest-corrections",
+	"profils": ["Checklist initiale"],
+	"wcag": ["3.3.3"],
+	"ID": "testMobID-7",
+	"IDorigin": "testMobID-7"
+}, {
+	"themes": "Touchscreen and interactions",
+	"title": "Ensure clickable areas are large enough.",
+	"type": ["Manual test"],
+	"tests": ["Identify clickable elements."],
+	"verifier": ["Clickable elements are sufficiently spaced apart.", "The touch area does not require excessive precision (recommended size: 44pt)."],

```

Comment:
~~~suggestion
	"verifier": ["Clickable elements are sufficiently spaced apart.", "The touch area does not require excessive precision (recommended size: <code>44pt</code>)."],
~~~